helix-term/src/commands.rs

```diff
@@ -2293,6 +2293,16 @@ fn open(cx: &mut Context, open: Open) {
         let mut text = String::with_capacity(1 + indent_len);
         text.push_str(doc.line_ending.as_str());
         text.push_str(&indent);
+
+        let token = doc
+            .language_config()
+            .and_then(|lc| lc.comment_token.as_ref())
+            .map(|tc| tc.as_ref());
```

Comment:
We might want to move this to a new method since this code is duplicated 3 times. Where do you suggest we put it?